### PR TITLE
added PnL to modal and market price to details

### PR DESCRIPTION
**Pull request by johnnieskywalker**
# [Title](https://clubhouseLink)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- item 1
  
## How to test 🧪
  <Please explain how to test your changes>
- step 1 ...
    
## Definition of done ✔️

- [ ] Acceptance criteria for each issue met
- [ ] Unit tests written where needed and passing
- [ ] End-to-end tests for happy path
- [ ] Documentation updated <When applicable>
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [ ] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

### CODE REVIEW

**Comment by johnnieskywalker** [comment]
in progress...
image

File: features/manageMultiplyVault/components/ManageMultiplyVaultView.tsx [hunk 1]
```diff
@@ -106,6 +107,8 @@ export function ManageMultiplyVaultContainer({
     }
   }, [])
 
+  manageVault.currentPnL= calculatePNL(vaultHistory, manageVault.netValueUSD)
```
Comment (johnnieskywalker) [inline features/manageMultiplyVault/components/ManageMultiplyVaultView.tsx:110]: not sure if this is nott a little code envy but calculations need history

**Comment by johnnieskywalker** [comment]
This is how it looks right now mobile version is not yet done
image

**Comment by johnnieskywalker** [comment]
Need to add mobile version as designed here : https://www.figma.com/file/kAPyiF2JpYre2pgxiYunEr/Oasis-Master?node-id=1001%3A25916

**Comment by johnnieskywalker** [comment]
so after some polishing it looks like this now 
image

**Comment by johnnieskywalker** [comment]
This is how it looks on mobile (iPhone 6/7/8) 
image


iPad
image


iPad Pro
image

Galaxy Fold
image

File: components/vault/VaultDetails.tsx [hunk 2]
```diff
@@ -320,14 +321,182 @@ export function VaultDetailsBuyingPowerModal({ close }: ModalProps) {
   )
 }
 
-export function VaultDetailsNetValueModal({ close }: ModalProps) {
+interface NetValueProps {
+  marketPrice: BigNumber | undefined
```
Comment (piekczyk) [inline components/vault/VaultDetails.tsx:325]: marketPrice?: BigNumber

Comment (johnnieskywalker) [reply components/vault/VaultDetails.tsx:325]: changed in whole file

**Review by piekczyk — APPROVED** [review]

**Pull request merged**